packages/plugin-interactive-tools/tests/colorizeUtils.test.ts

```diff
@@ -0,0 +1,23 @@
+import {Configuration}       from '@yarnpkg/core';
+import {xfs, PortablePath}   from '@yarnpkg/fslib';
+
+import {colorizeVersionDiff} from '../sources/colorizeUtils';
+
+async function initializeConfiguration<T>(value: {[key: string]: any}, cb: (dir: PortablePath) => Promise<T>) {
```

Comment:
copied from `Cofiguartion.test.ts`, is there a better way to create a configuration object?